tests/zkevm/test_worst_bytecode.py

```diff
@@ -27,7 +27,7 @@
 REFERENCE_SPEC_VERSION = "TODO"
 
 MAX_CONTRACT_SIZE = 24 * 1024  # TODO: This could be a fork property
-OPCODE_GAS_LIMIT = 100_000
+OPCODE_GAS_LIMIT = 36_000_000
```

Comment:
Here's the main contention point I think.

Ideally, I would like to put `Environment().gas_limit`. But I'm already using the `--block-gas-limit` for the actual gas limit of the block, not the attack.

If the conclusion is that we can hardcode the gas limit at the test level to 100GGas, then I could use `Environment().gas_limit` here without problems, and it will fit what it is done in #1382

Reply:
I think this is ok, `Environment().gas_limit` is just a recommendation to constrain tests to not go above this limit to maintain the testing realistic, but if the setup goes above that limit it should be ok.

If we go this route, one consideration is that the test is not going to be executable via `uv run execute`, but we could introduce a fixture that lets the test know the mode it is currently running in and limit the gas consumed by its setup. Just an idea for now, such a fixture does not exist yet.

Reply:
Cool, so I'll change this `OPCODE_GAS_LIMIT` `36m` to `Environment().gas_limit` (so behaves coherently with whatever `--block-gas-limit` is provided via CLI) and hardcode the 100GGas for gas limit.

Reply:
Done! 

Let's wait to merge #1578 first, and then I'll rebase this PR and we can do a final check before merging.